### DISCUSSION TITLE
Question about PayPal language

### DISCUSSION THREAD

**Discussion by justyasen** [Adyen/adyen-react-native#322] [Q&A]
Hello, 

We are passing a french locale(fr-FR) to the Adyen call that handles opening the PayPal link, however the PayPal page is still in English.

My question is - is it possible to get the PayPal page in English - and if so - are we supposed to pass a specific field to the Adyen call to make sure that happens or how would you suggest we go at it? 

Thanks, 
Yasen

**Comment by camil-adyen** [comment]
Hi @justyasen, 

Are you passing the shopperLocale in the /payments request? Doing so will return a URL in the response that links to the PayPal page in the specified language. If that doesn't work, please try opening the link in a private window. 

Best,
Camil
